package.json

```diff
@@ -50,16 +50,19 @@
     "util:copy-icons": "cpy \"./node_modules/@esri/calcite-ui-icons/js/*.json\" \"./src/components/icon/assets/icon/\" --flat",
     "util:deploy-next": "npm run util:prep-next && npm run util:publish-next",
     "util:deploy-next-from-ci": "ts-node --esm support/deployNextFromCI.ts",
-    "util:is-next-deployable": "ts-node --esm support/isNextDeployable.ts",
+    "util:generate-t9n-types": "ts-node --esm support/generateT9nTypes.ts",
```

Comment:
generateT9nTypes file is missing in this PR.

Reply:
That along with other files (including the main t9n one) 🤦. Thanks for catching this!